build.json

```diff
@@ -65,6 +69,6 @@
     "output_wrapper": "(function(){%output%}).call(window);",
     "export_local_property_definitions": true,
     "use_types_for_optimization": true,
-    "manage_closure_dependencies": true
+    "manage_closure_dependencies": false
```

Comment:
If this param is ``true`` closure compiler fails saying that
> ERR! compile node_modules/openlayers/externs/olx.js:1263: ERROR - Bad type annotation. Unknown type ol.control.ScaleLineUnits
ERR! compile  *     units: (ol.control.ScaleLineUnits|string|undefined)}}
ERR! compile                ^
ERR! compile 
ERR! compile node_modules/openlayers/externs/olx.js:1303: ERROR - Bad type annotation. Unknown type ol.control.ScaleLineUnits
ERR! compile  * @type {ol.control.ScaleLineUnits|string|undefined}
ERR! compile           ^
ERR! compile 
ERR! compile node_modules/openlayers/externs/olx.js:1794: ERROR - Bad type annotation. Unknown type ol.format.IGCZ
ERR! compile  * @typedef {{altitudeMode: (ol.format.IGCZ|undefined)}}
ERR! compile                              ^
ERR! compile 
ERR! compile 
ERR! compile node_modules/openlayers/externs/olx.js:1802: ERROR - Bad type annotation. Unknown type ol.format.IGCZ
ERR! compile  * @type {ol.format.IGCZ|undefined}
ERR! compile           ^